AgentDiscoveries-Frontend/app/sass/styles.scss

```
@@ -85,3 +86,13 @@ body {
 #report-input {
    resize: none;
 }
+
+input{
+  color:black;
+
+}
+
+tbody{
+  color:black;
+  background-color: grey;

```

```suggestion
  background-color: #ededed;
```
No problem here really, just looks a little cleaner with a lighter grey